### DISCUSSION TITLE
Suggested new notifier

### DISCUSSION THREAD

**Discussion by daphilli224** [zencart/zencart#6569] [General]
Please add a new notifier in ot_group_pricing.php, calculate_deductions function, in the next zencart release to allow an observer to override the discount amount for guest customers. See One-Page Checkout Support forum thread post #2876. Thank you!

**Comment by scottcwilson** [comment]
Can you PR @daphilli224 ?

**Comment by daphilli224** [comment]
Scott, I don’t feel capable of generating a PR with all the GitHub jargon. The code to add is in the referenced forum post and was generated by lat9. Sorry.
Dave

**Reply by lat9** [reply]
No problem, Dave, I'll get it!

**Comment by daphilli224** [comment]
Thank you! About the only thing I can do on GitHub is write up an issue or, as of today, initiate a discussion. I would have no idea what branch or repository to use for a new PR. And I assume it would be a draft PR?

**Comment by scottcwilson** [comment]
This repo, master branch.  Watch how @lat9 does it and maybe you'd like to give it a whirl next time! 

We have some docs: 
https://docs.zen-cart.com/dev/contributing/

**Comment by drbyte** [comment]
Resolved in #6573 and #6572
